src/components/header.js

```diff
@@ -1,7 +1,129 @@
-import React from "react";
+import React, { useState } from "react";
+import LanguageSwitcher from "./LanguageSwitcher";
+import {
+  ExtendedNav,
+  GovBanner,
+  Header as HeaderUSWDS,
+  Link,
+  Menu,
+  NavDropDownButton,
+  NavMenuButton,
+  Search,
+  Title,
+} from "@trussworks/react-uswds"; // TODO: read through documentation of each component to see if used correctly
+import { withTranslation } from "react-i18next";
 
-const Header = () => {
-  return <div>Header</div>;
+const Header = ({ t, i18n }) => {
+  const govBannerLang = {
+    /* TODO: might want to keep in a separate file */
+    en: "english",
+    es: "spanish",
+  };
+
+  const [expanded, setExpanded] = useState(false);
+  const onClick = () => setExpanded((prvExpanded) => !prvExpanded);
+  const [isOpen1, setIsOpen1] = useState([false]);
+  const [isOpen2, setIsOpen2] = useState([false]);
+
+  const testMenuItemsOne = [
+    <Link href="#linkOne" key="one">
```

Comment:
I believe we should use 'to' not 'href'. Also let's look into whether we should use Link from react-uswds or from Gatsby

Reply:
Thanks, at the time I was wondering because I realized it allowed both and was trying to figure out if it would make a difference. Also, I was going to mention that! Let's bring it up to Rapi.